lib/render/mayaToHydra/plugin.cpp

```diff
@@ -0,0 +1,123 @@
+//
+// Copyright 2019 Luma Pictures
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include <maya/MFnPlugin.h>
+
+#include "renderGlobals.h"
+#include "renderOverride.h"
+#include "usdPreviewSurface.h"
+#include "viewCommand.h"
+
+#include <stdlib.h>
+
+#include <pxr/base/plug/plugin.h>
+#include <pxr/base/plug/registry.h>
+#include <pxr/base/tf/envSetting.h>
+
+#include "../../usd/hdMaya/adapters/adapter.h"
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+TF_DEFINE_ENV_SETTING(
+    MTOH_ENABLE_USD_PREVIEW_SURFACE_NODE, true,
+    "Enables the registration of the UsdPreviewSurface node."
+    "This is not required with newer version of usdMaya.");
```

Comment:
Good catch - this code is now never needed, since it's now in the same repo as the pixar mayaUsd plugin, we can assume that it's always there.  All related code was removed in:

https://github.com/Autodesk/maya-usd/pull/231/commits/43a9bb45e73871cc9f5fe58ec830a0fb5ab05ab4